addons/web/static/src/views/view.scss

```diff
@@ -50,6 +50,27 @@
     }
 }
 
+// the nocontent helper in fields specially for hr employee view form
```

Comment:
As stated in your comment... this is specific to "HR" module... So why putting it in `web` ? :thinking: